horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
Is there a way to exclude blocks of code from certain checks? I prefer the original layout, especially when lots of lines are aligned in a tabular-like style.